attack_run.sh

```diff
@@ -3,9 +3,18 @@
 set -u
 
 export QUAY_URL=<URL>
+export UUID=$(uuidgen)
+export DURATION=120
 export PARALLELISM=10
-export NUM_USERS=1000
-export RATE=10
+export NUM_USERS=10
+export RATE=40
+export PREFIX=perf-test
+export ES=search-cloud-perf-lqrf3jjtaqo7727m7ynd2xyt4y.us-west-2.es.amazonaws.com
```

Comment:
Do you want this in your public Github repo?

Reply:
yeah. It is everywhere. No problem!